test

```diff
@@ -60,3 +60,15 @@ fi
 # See https://github.com/bitnami/minideb/issues/16
 desc "Check that we can install mysql-server (#16)"
 test install_packages $MYSQL_PACKAGE
+
+function shadow_check() {
+    local path=$1
+    shift
```

Comment:
I think this command here don't provide any functionality as the function is always called with one parameter only.
Even the `local path=$1` could be removed and use just `$1`

Reply:
> Even the local path=$1 could be removed and use just $1
Yeah, I like the self-documenting nature of naming the parameter.

Reply:
`shift` removed, please take another look.